dockerfiles/centos/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM centos:centos7.9.2009
+
+# Install OS dependencies
+RUN yum update -y && \
+    yum install -y \
+        libXext-devel \
+        libXi-devel \
+        mesa-libGL-devel \
+        mesa-dri-drivers \
+        xorg-x11-server-Xvfb \
+        wget \
+        which \
+        make \
+        scl-utils \
+        centos-release-scl
+
+# Broken into two steps so scl gpg key is available from above
+RUN yum install -y \
+        devtoolset-8-gcc \
+        devtoolset-8-gcc-c++
+
+# Install nvm with node and npm
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 12.21.0
+RUN mkdir /usr/local/nvm
+RUN curl -sL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash \
+    && . $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH      $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
+# Install application
+WORKDIR /opt/app
+
+COPY . .
+
+RUN source scl_source enable devtoolset-8 && \
```

Comment:
What is `source scl_source enable devtoolset-8` for? Maybe add an inline comment.

Reply:
Right, I was surprised at how much c/c++ dev tooling we needed for building JS packages as well.